geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/ConverterIngestProcessor.scala

```diff
@@ -142,7 +147,10 @@ trait ConverterIngestProcessor extends AbstractGeoIngestProcessor {
 
       val converter = converters.borrowObject()
       try {
-        val ec = converter.createEvaluationContext(EvaluationContext.inputFileParam(name))
+        val globalParams =
+          EvaluationContext.inputFileParam(name) ++
+              (if (exposeAttributes) { file.getAttributes.asScala -- Attributes.all } else { Map.empty })
```

Comment:
Does the -- Attributes.all strip out the datastore params?  Like the password?  

If so, that's good catch.:)

Reply:
no, that just strips out the geomesa.convert and geomesa.sft configs that may be present. the datastore params aren't exposed as flow file attributes, these would be other per-file metadata (like timestamps or correlation fields)